servers/rendering/shader_language.cpp

```diff
@@ -9383,6 +9383,9 @@ Error ShaderLanguage::_parse_shader(const HashMap<StringName, FunctionInfo> &p_f
 										_set_error(RTR("Array size mismatch."));
 										return ERR_PARSE_ERROR;
 									}
+								} else {
+									_set_expected_error("(,{");
```

Comment:
```suggestion
									_set_expected_error("(");
```

`curly` is handled by previous `if` + this is incorrect pass of multiple arguments to `_set_expected_error`.

Reply:
Thanks!